src/panels/lovelace/components/hui-image.js

```diff
@@ -15,6 +15,10 @@ class HuiImage extends LocalizeMixin(PolymerElement) {
   static get template() {
     return html`
       <style>
+        :host {
+          overflow-y: hidden;
```

Comment:
Why?

Reply:
If this is because of it being placed in `entity-picture`, the CSS should be added to `entity-picture`. We should never add CSS to web components because it helps styling it in a parent. The parent should contain the styling instead.

Reply:
This became apparent because of picture-elements, but I think it is the correct behavior in general. If the parent has a static height I feel like the expectation is that the image won't overflow. But the only place this is a factor right now is picture-elements, so I'll move this css there and reassess if it comes up other places too